src/StoreApi/Routes/V1/CartUpdateCustomer.php

```diff
@@ -62,6 +60,51 @@ public function get_args() {
 		];
 	}
 
+	/**
+	 * Validate address params now they are populated.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 * @param array            $billing Billing address.
+	 * @param array            $shipping Shipping address.
+	 * @return \WP_Error|true
+	 */
+	protected function validate_address_params( $request, $billing, $shipping ) {
```

Comment:
This is based on the validation code in WP core. It runs the original validation functions on the addresses and returns a WP_Error if there are issues with fields.